db/migrate/20180118134516_create_digest_runs.rb

```diff
@@ -0,0 +1,12 @@
+class CreateDigestRuns < ActiveRecord::Migration[5.1]
+  def change
+    create_table :digest_runs do |t|
+      t.date :date, null: false
+      t.datetime :starts_at, null: false
+      t.datetime :ends_at, null: false
```

Comment:
I wonder if it's better to use a word like `until` here to suggest that it's a `<` not a `<=` relation.

Reply:
I'm easy either way... I'll change unless anyone else has any opinions.

Reply:
Actually... no I won't. `until` is a reserved word (as was `end` when I originally named them `start` and `end` 🤕 )

Reply:
Hmm, oh yeah... I'll leave it as it is, we can see if it's confusing.